src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts

```diff
@@ -19,21 +19,18 @@ describe('Target Page visualization boxes', () => {
         await popupPage.gotoAdhocPanel();
     });
 
-    beforeEach(async () => {
-        await popupPage.disableAllToggles();
-        await targetPage.waitForVisualizationBoxesToDisappear();
-    });
-
     afterAll(async () => {
         if (browser) {
             await browser.close();
             browser = undefined;
         }
     });
 
-    const adhocTools = ['Automated checks', 'Headings', 'Landmarks'];
+    const adhocTools = ['Automated checks', 'Headings'];
```

Comment:
It would be good to leave a comment here describing why we're only testing some of the tools